trace/opencensus/trace/exporters/transports/background_thread.py

```diff
@@ -0,0 +1,146 @@
+# Copyright 2017, OpenCensus Authors
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import atexit
+import threading
+import time
+
+from six.moves import queue
+from six.moves import range
+
+_DEFAULT_GRACE_PERIOD = 5.0  # Seconds
+_WAIT_PERIOD = 3.0  # Seconds
+_DEFAULT_MAX_BATCH_SIZE = 2
+_WORKER_THREAD_NAME = 'opencensus.trace.Worker'
+_WORKER_TERMINATOR = object()
+
+
+class _Worker(object):
+    def __init__(self, exporter, grace_period=_DEFAULT_GRACE_PERIOD,
```

Comment:
The google-cloud-python code which inspired this is well documented, e.g. https://github.com/GoogleCloudPlatform/google-cloud-python/blob/master/logging/google/cloud/logging/handlers/transports/background_thread.py#L65  .  Can you write/copy doc comments into this code to match?

It would be nicest if we could both use the same code, but I don't think that's feasible right now.

Reply:
Done.